testsuite/tests/conftest.py

```diff
@@ -828,3 +829,19 @@ def prometheus(testconfig, openshift):
     prometheus_url = protocol + routes[0]['spec']['host']
 
     return PrometheusClient(prometheus_url)
+
+
+@pytest.fixture(scope='session')
+def stripe():
+    """Stripe API"""
```

Comment:
will be this ever needed outside of UI? Actually it is misleading question. Move it to ui/constest.py